### ISSUE TITLE
add nameOverride, fullnameOverride to the helm chart 

### ISSUE THREAD

**Issue by ndallavalentina** [vmware-tanzu/helm-charts#464]
I would suggest adding those values to the helm chart  please:

```
nameOverride: ""
fullnameOverride: ""
```
then I can rewrite the name of it

**Comment by qiuming-best** [comment]
@ndallavalentina I've transferred it to this repo

@jenting what's your opinion on this requirement?

**Comment by jenting** [comment]
@ndallavalentina @qiuming-best 

I think the chart supports these two fields already
https://github.com/vmware-tanzu/helm-charts/blob/45c90794d921fee8a975509fb3e7baf1df2d9e6e/charts/velero/templates/_helpers.tpl#L6
https://github.com/vmware-tanzu/helm-charts/blob/45c90794d921fee8a975509fb3e7baf1df2d9e6e/charts/velero/templates/_helpers.tpl#L15

**Comment by ndallavalentina** [comment]
that's correct but I can't override with values.yaml. It will automatically set release-*
